src/index.jsx

```diff
@@ -69,22 +56,41 @@ function configure() {
   };
 }
 
-apiClient.ensurePublicOrAuthenticationAndCookies(
-  window.location.pathname,
-  (accessToken) => {
-    const { store, history } = configure();
+function initialize(accessToken) {
+  const { store, history } = configure();
+
+  if (accessToken) {
+    ReactDOM.render(<App store={store} history={history} />, document.getElementById('root'));
+    identifyAuthenticatedUser(accessToken.userId);
+    sendPageEvent();
+  } else {
+    // This should never happen, but it does sometimes.
```

Comment:
This section has changed.  Now we throw an error which gets caught and handled below.  We used to render the error page and log to new relic directly in here.